nl/mail/chrome/messenger/preferences/applications.dtd

```diff
@@ -24,14 +24,14 @@ entities below were automatically added by l10nmerge
 -->
 
 <!ENTITY enableCloudFileAccountOffer.mb "MB">
-<!ENTITY authRequired.button.label        "Authorize">
+<!ENTITY authRequired.button.label        "Autoriseren">
 <!ENTITY authRequired.button.accesskey    "U">
 <!ENTITY addCloudFileAccount.accesskey    "A">
-<!ENTITY removeCloudFileAccount.label     "Remove">
-<!ENTITY addCloudFileAccount.description  "Add a new online storage service">
-<!ENTITY authRequired.description         "Your authorization is required in order to see the settings for this storage service.">
-<!ENTITY addCloudFileAccount.label        "Add">
+<!ENTITY removeCloudFileAccount.label     "Verwijderen">
+<!ENTITY addCloudFileAccount.description  "Voeg een nieuwe online opslag service toe">
+<!ENTITY authRequired.description         "Uw autotisatie is nodig om de instellingen voor deze opslag service te zien.">
```

Comment:
CORRECTION NEEDED, it should be: "Uw autorisatie is nodig om de instellingen voor deze opslagdienst te zien."